packages/atomic/src/utils/accessibility-utils.tsx

```diff
@@ -149,3 +149,9 @@ export function getFirstFocusableDescendant(
   }
   return null;
 }
+
+function getSearchUIDFromState(bindings: any): string {
```

Comment:
I know this `any` (and the other ones that have warnings in this PR) is temporary, but would `unknown` work instead and remove the warnings?